src/Illuminate/Validation/Concerns/FormatsMessages.php

```diff
@@ -320,6 +320,10 @@ public function getDisplayableValue($attribute, $value)
             return $line;
         }
 
+        if (is_bool($value)) {
+            return $value ? 'true' : 'false';
```

Comment:
what about other languages?

Reply:
good point. I'm not sure if it should be translated as it's a reference to the data type

Reply:
@taylorotwell would you accept a PR that makes this strings translatable?